redaxo/src/core/lang/de_de.lang

```diff
@@ -230,6 +230,16 @@ sql_unable_to_create_database = Die angegebene Datenbank konnte nicht angelegt w
 sql_unable_to_find_database = Die angegebene Datenbank konnte nicht gefunden werden!
 sql_unable_to_open_database = Die angegebene Datenbank konnte nicht geöffnet werden!
 
+setup_security = Sicherheit
+
+https_activate_redirect_for = Automatische HTTPS-Umleitung
+https_disable = deaktiveren
+https_only_frontend = nur für Frontend aktivieren
+https_only_backend = nur für Backend aktivieren
+https_activate = für Frontend und Backend aktiveren
```

Comment:
Vielleicht die Keys sogar generischer benennen, da der Text ja auch allgemein ist. Sodass man sie zukünftig auch für andere Optionen nehmen kann? Also `activate_frontend_and_backend` oder irgendwie so.
Oder findet ihr so spezifische Keys (hier `https_...`) sinnvoller?

Reply:
ja und nein, manchmal bekommt man mit den vorhandenen Texten keine sinnvolle Grammatik heraus.
Aber in diesem Fall hast du Recht, die Texte sind allgemein und ließen sich an anderen Stellen auch gut nutzen.

Reply:
Wenn wir diese texte generisch machen wollen, sollten es „volle sätze werden“ inkl einem platzhalter für das subjekt, oder?